### PR TITLE
Add form styles to global stylesheet

### PR DESCRIPTION
**Pull request by kbratko**

### CODE REVIEW

File: src/scss/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+$fullWidth: 100%;
+$halfWidth: $fullWidth/2;
+
+@mixin clearfix {
```
Comment (jehoshua02) [inline src/scss/_variables.scss]: Not a variable.

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  &::after {
+    clear: both;
+    content: "";
+    display: table;
+  }
+}
+
+@mixin minMedium() {
```
Comment (jehoshua02) [inline src/scss/_variables.scss]: Also not a variable.

File: src/scss/_variables.scss (hunk 1 above)
Comment (janiv) [inline src/scss/_variables.scss]: To clarify, Josh's comments he'd prefer these mixins moved to their own mixin file.

File: src/scss/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+$fullWidth: 100%;
+$halfWidth: $fullWidth/2;
```
Comment (jehoshua02) [inline src/scss/_variables.scss]: Instead of a variable for these widths, I'd just use the literals `100%` and `50%` and get rid of the variables.

**Comment by jehoshua02** [comment]
@kbratko It looks as though the form styles were copied and pasted from the cars page. I would have liked to see an iterative approach, adding one piece at a time, starting with the smallest pieces, making sure each piece fits neatly into ksl-styles and will be easy to reuse. For example, we should add styles for `.text-input` before we add styles for `.form`, the `.text-input` styles should not depend on `.form`, and the docs should show what `.text-input` looks like by itself.

**Comment by jehoshua02** [comment]
@kbratko There are media queries in the form styles. I tend to think that media queries, unless carefully crafted, are not easy to reuse. They are typically page specific and concerned with layout. I think it's too early to have layout styles in ksl-styles.

**Comment by kbratko** [comment]
I confused a bit. Where I should place styles for form layout and fields alignment? Leave them in ksl-cars?

**Comment by jehoshua02** [comment]
@kbratko For now, yes, I'd say. Let's build up ksl-styles from the smaller pieces first.

File: src/scss/styles.scss [hunk 4]
```diff
@@ -1,5 +1,9 @@
 @import './ksl-icons/styles.css';
 @import './fonts/styles.css';
+
+@import tools/clearfix;
+@import tools/mediaQueries;
```
Comment (jehoshua02) [inline src/scss/styles.scss]: Let's quote them like the rest.

**Comment by kbratko** [comment]
@jehoshua02  I've added changes but I cant build them. May be you can contact me and help me with build. I've pushed what I have. Please take a look on it.

**Comment by kbratko** [comment]
@jehoshua02  I've made a push. So let me know if I should do some more.

**Comment by kbratko** [comment]
@jehoshua02  I reviewed your changes. You've done a great work. I like an idea to use some common padding variables. I've started creating global styles in another branch for responsive grids using your comments and tips.

**Comment by janiv** [comment]
@kbratko For the responsive grid, I'd prefer using https://github.com/ericam/susy rather than something we build in house. We'll need to use at least libsass 3.1 and then v.2.2.0-rc2 of susy.

**Comment by kbratko** [comment]
@janiv  I'll learn it. Thanks

**Comment by jehoshua02** [comment]
@janiv I'd like to build on top of this so I'm merging now. You're still welcome to review. I'll fix anything you find in another pull request.

**Pull request merged**